Whitepapers/THORChain/whitepaper-en.md

```diff
@@ -127,7 +127,7 @@ A self-amending forkless consensus algorithm for THORChain.
 ### Centralized and Decentralized Exchanges
 
 **Exchanges**. One of the most important advents of the cryptocurrency economy is the crypto exchange.  While digital assets are useful on their own, exchanges are needed so that creators, consumers, investors, and traders can transfer these assets to one other openly and freely.  As digital assets reach market equilibrium prices, all market participants benefit from clearing markets that are otherwise either unavailable or not at true equilibrium due to interference from governments, regulators, or logistical barriers.
-Venture capitalists, service providers, employees, founders, and other company stakeholders benefit from freely available liquidity that is not locked up in arcane or encumbering pre-IPO restrictions.  This is true for tokens regardless of whether they are asset-backed, service-backed, or infrastructure-backed. In the asset-backed case case, cryptocurrency exchanges allow for easier transactions of securities that are currently more costly in terms of time and money due to reconciliation requirements that are natively on the blockchain and otherwise handled by back-office workers.  
+Venture capitalists, service providers, employees, founders, and other company stakeholders benefit from freely available liquidity that is not locked up in arcane or encumbering pre-IPO restrictions.  This is true for tokens regardless of whether they are asset-backed, service-backed, or infrastructure-backed. In the asset-backed case, cryptocurrency exchanges allow for easier transactions of securities that are currently more costly in terms of time and money due to reconciliation requirements that are natively on the blockchain and otherwise handled by back-office workers.  
```

Comment:
@okwme could you add an extra space at the beginning of the 3rd sentence? 

```suggestion
Venture capitalists, service providers, employees, founders, and other company stakeholders benefit from freely available liquidity that is not locked up in arcane or encumbering pre-IPO restrictions.  This is true for tokens regardless of whether they are asset-backed, service-backed, or infrastructure-backed.  In the asset-backed case, cryptocurrency exchanges allow for easier transactions of securities that are currently more costly in terms of time and money due to reconciliation requirements that are natively on the blockchain and otherwise handled by back-office workers.  
```